sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_user_agent.py

```diff
@@ -2,7 +2,6 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-import platform
 from ._version import VERSION
 
 USER_AGENT = "keyvault-certificates/{}".format(VERSION)
```

Comment:
Please forgive me for being nitpicky 😇 but this is misnamed now that it isn't the user agent. Autorest and core have apparently settled on `sdk_moniker` for strings like this, maybe this should be `_moniker.py` and `SDK_MONIKER`?

Reply:
That makes sense. Changing now!